src/content/docs/developer-guide/hint-join-order.md

```diff
@@ -8,33 +8,36 @@ the join strategy. See below for the available hints for Kùzu 0.5.0 and upward.
 
 ## Join order hint
 
-In certain cases, Kùzu might generate a sub-optimal join order when the obtained statistics are
-inaccurate at the optimization stage of a large graph query. We provide join order hints to allow
-users to enforce a specific join order that bypasses Kùzu's optimizer.
-
-As an example, for the following query, the optimizer will always compile a query plan that scans
-from `a` due to the index lookup.
-
+Consider the following query
 ```cypher
-MATCH (a)-[e]->(b)
-WHERE a.ID = 0
+MATCH (a:person)-[e:LivesIn]->(b:City)
 RETURN *;
 ```
+The system optimizer has its freedom to generate a query plan that find all edges from `a` to `b` using forward adjcency list or from `b` to `a` using backward adjcency list. Both query plan are semantically equivalent and generate the same query result. There is, however, a potential performance difference because `a` and `b` may have different cardinalities.
```

Comment:
```suggestion
The system optimizer has its freedom to generate a query plan that finds all edges from `a` to `b` using forward adjacency list or from `b` to `a` using a backward adjacency list. Both query plans are semantically equivalent and generate the same query result. There is, however, a potential performance difference because `a` and `b` may have different cardinalities.
```